include/bluetooth/mesh/main.h

```diff
@@ -272,6 +286,23 @@ int bt_mesh_input_string(const char *str);
  */
 int bt_mesh_input_number(uint32_t num);
 
+/** @brief Register a callback to retrieve a local public key.
+ *
+ * This function needs to be used when @ref BT_MESH_PUB_KEY_OOB is set.
+ * The retrieved public key has to be shared with a Provisioner using
+ * any available OOB technology.
+ *
+ * A new key is generated after each provisioning attempt.
+ *
+ * Used only by Provisionee.
```

Comment:
According to https://github.com/zephyrproject-rtos/zephyr/pull/28457#discussion_r501068834 .
Should it be:
```suggestion
 * Used only by device provisioning.
```